backporter.jl

```diff
@@ -17,7 +17,9 @@ import HTTP
 # Settings #
 ############
 
-BACKPORT = "1.9"
+_BACKPORT_DEFAULT = "1.9"
+
+BACKPORT = strip(get(ENV, "BACKPORTER_SCRIPT_TARGET_VERSION", _BACKPORT_DEFAULT))
```

Comment:
Seems far too long to be useful
```suggestion
BACKPORT = strip(get(ENV, "BACKPORT_VERSION", _BACKPORT_DEFAULT))
```

Reply:
Done!